components/UI/side-drawer/side-drawer.module.scss

```diff
@@ -11,13 +11,13 @@
   transition: transform 0.3s ease-out;
   border-radius: 0 8px 8px 0;
 }
-.logoName{
+.logoName {
```

Comment:
Is this happening due to prettier or some extension?
I don't observe any code changes here

Reply:
Might so.